### PR TITLE
pawnKind combat skill range tweaks

### PR DESCRIPTION
**Pull request by ItsSamGray**
## Changes

- Added and tweaked skill range values for vanilla and modded pawnKinds
- Fixed the Pirate Captain role bonuses from VFE-Pirates
- Some housekeeping for the combat oriented ideology abilities

## Reasoning

- Vanilla mercenary gunners and snipers had a shooting skill range of 4~20. Imperial pawns had various combat skill ranges as well which sparked the thought of tweaking and adding the melee and shooting skill range to certain high tier faction pawns to make them more reliable in their combat role.

## Testing

Check tests you have performed:
- [x] Compiles without warnings
- [x] Game runs without errors

### CODE REVIEW

File: Patches/Core/PawnKindDefs_Humanlike/PawnKinds.xml [hunk 1]
```diff
@@ -807,4 +955,19 @@
...
 
+  <Operation Class="PatchOperationConditional">
+   <xpath>/Defs/PawnKindDef[defName="StrangerInBlack"]/skills</xpath>
+   <nomatch Class="PatchOperationAdd">
+     <xpath>/Defs/PawnKindDef[defName="StrangerInBlack"]</xpath>
+     <value>
+       <skills>
+         <li>
+           <skill>Shooting</skill>
+           <range>8~14</range>
```
Comment (N7Huntsman) [inline Patches/Core/PawnKindDefs_Humanlike/PawnKinds.xml:966]: Way too high. Don't care if they're supposed to be "saving" the colony, that doesn't prevent them being a fuck-up.

File: Patches/Vanilla Factions Expanded - Ancients/PawnKinds_Spacer.xml [hunk 2]
```diff
@@ -29,25 +29,23 @@
           </value>
         </li>
 
-        <li Class="PatchOperationAddModExtension">
-          <xpath>/Defs/PawnKindDef[defName="VFEA_AncientSoldierTwoAbilities" or defName="VFEA_PlayerAncientSoldierOneAbility"]</xpath>
-          <value>
-            <li Class="CombatExtended.LoadoutPropertiesExtension">
```
Comment (N7Huntsman) [inline Patches/Vanilla Factions Expanded - Ancients/PawnKinds_Spacer.xml:35]: Why'd you remove their ammo???

**Review by N7Huntsman — DISMISSED** [review]
I'm not in support of most of these changes, frankly. At most, I could get behind increasing the minimum for a select few elite-level pawns, but not fodder like Town Guards.

You're not fighting the Ancient soldiers to any large degree (and those few that will try to fight will have cryosickness, anyway), so I don't see the point in raising theirs.

File: Patches/Vanilla Factions Expanded - Ancients/PawnKinds_Spacer.xml (hunk 2 above)
Comment (ItsSamGray) [reply Patches/Vanilla Factions Expanded - Ancients/PawnKinds_Spacer.xml:35]: the patch was borked and had duplicate entries for the VFEA_PlayerAncientSoldierOneAbility, the VFEA_AncientSoldierTwoAbilities was added to the first patch's address

File: Patches/Core/PawnKindDefs_Humanlike/PawnKinds.xml (hunk 1 above)
Comment (ItsSamGray) [reply Patches/Core/PawnKindDefs_Humanlike/PawnKinds.xml]: Done.

**Review by N7Huntsman — APPROVED** [review]

**Pull request merged**